suite-native/device/src/components/ConfirmOnTrezorBottomSheet.tsx

```diff
@@ -1,13 +1,18 @@
 import { BottomSheet, Button, VStack, Box, Text } from '@suite-native/atoms';
 import { Translation } from '@suite-native/intl';
+import { ReactNode } from 'react';
```

Comment:
fyi: there are some linting errors not catched by checks

Reply:
Good catch. Fixed (already squashed).